src/java.management/share/classes/javax/management/loading/MLet.java

```diff
@@ -1,1294 +0,0 @@
-/*
- * Copyright (c) 1999, 2022, Oracle and/or its affiliates. All rights reserved.
- * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
- *
- * This code is free software; you can redistribute it and/or modify it
- * under the terms of the GNU General Public License version 2 only, as
- * published by the Free Software Foundation.  Oracle designates this
- * particular file as subject to the "Classpath" exception as provided
- * by Oracle in the LICENSE file that accompanied this code.
- *
- * This code is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
- * version 2 for more details (a copy is included in the LICENSE file that
- * accompanied this code).
- *
- * You should have received a copy of the GNU General Public License version
- * 2 along with this work; if not, write to the Free Software Foundation,
- * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
- *
- * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
- * or visit www.oracle.com if you need additional information or have any
- * questions.
- */
-
-package javax.management.loading;
-
-import com.sun.jmx.defaults.ServiceName;
-
-import com.sun.jmx.remote.util.EnvHelp;
-
-import java.io.Externalizable;
-import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.ObjectInput;
-import java.io.ObjectInputStream;
-import java.io.ObjectOutput;
-import java.lang.reflect.Constructor;
-import java.net.MalformedURLException;
-import java.net.URL;
-import java.net.URLStreamHandlerFactory;
-import java.nio.file.Files;
-import java.nio.file.StandardCopyOption;
-import java.security.AccessController;
-import java.security.PrivilegedAction;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.lang.System.Logger.Level;
-import java.util.Map;
-import java.util.Set;
-import java.util.StringTokenizer;
-
-import javax.management.InstanceAlreadyExistsException;
-import javax.management.InstanceNotFoundException;
-import javax.management.MBeanException;
-import javax.management.MBeanRegistration;
-import javax.management.MBeanRegistrationException;
-import javax.management.MBeanServer;
-import javax.management.NotCompliantMBeanException;
-import javax.management.ObjectInstance;
-import javax.management.ObjectName;
-import javax.management.ReflectionException;
-
-import static com.sun.jmx.defaults.JmxProperties.MLET_LIB_DIR;
```

Comment:
Do those 2 statics in JmxProperties also need to be removed?

Reply:
Yes good question - there are a couple of other definitions, they will be removed later if not now.

I was hesitant and was thinking maybe they should be removed later.  But looking again I find no other references to these, they are not part of the documented interface, so I should update this to remove them now.


src/java.management/share/classes/com/sun/jmx/defaults/JmxProperties.java
MLET_LIB_DIR only used by MLet.java
MLET_LOGGER_NAME used in same file to define MLET_LOGGER 
MLET_LOGGER used by MLet.java and MLetParser.java

JMX_INITIAL_BUILDER defined above MLET_LIB_DIR has an incorrect name in the comment.


src/java.management/share/classes/com/sun/jmx/defaults/ServiceName.java
public static final String MLET = "type=MLet"; only used by MLet.java